src/Query/index.ts

```diff
@@ -15,18 +15,18 @@ export class Query {
   terms: Set<string>;
   phrases: string[];
   rawQuery: string;
-  filters: Filter<Document>;
+  // filters: Filter<Document>;
 
   /**
    * Create a new query.
    * @param {string} queryString The query to parse
    */
-  constructor(queryString: string, filters?: Filter<Document>) {
+  constructor(queryString: string) {
     console.log('Query parsing: ' + queryString);
     this.terms = new Set();
     this.phrases = [];
     this.rawQuery = queryString;
-    this.filters = filters || {};
+    // this.filters = filters || {};
 
```

Comment:
Should these be deleted? Or if left in, maybe leave a comment explaining?

Reply:
🤦 thanks for the catch! removed